author.php

```diff
@@ -1,22 +1,24 @@
 <?php
-	get_header();
-	$author_image = get_avatar( get_queried_object()->ID, 'squared' );
+get_header();
+$author_image_url = get_avatar_url(get_queried_object()->ID, 'squared');
 ?>
-<header class="page-header">
+<header class="page-header  bg-grey">
+	<?php
+	if (function_exists('yoast_breadcrumb')) {
+		yoast_breadcrumb('<p id="breadcrumbs">', '</p>');
+	}
+	?>
 	<div class="container">
-		<div class="columns is-centered">
-			<?php if ( ! empty( $author_image ) ) : ?>
-				<div class="column is-1">
-					<?php echo $author_image; ?>
+		<div class="columns m-0 is-centered">
+			<?php if (!empty($author_image_url)) : ?>
+				<div class="column is-4">
+					<img alt="" src="<?php echo $author_image_url ?>" class="avatar avatar-96 full-height full-width photo p-3 bg-white" loading="lazy">
 				</div>
 			<?php endif; ?>
 			<div class="column">
-				<h2><?php echo CC_Site::page_title(); ?></h2>
-				<?php
-				if ( function_exists( 'yoast_breadcrumb' ) ) {
-					yoast_breadcrumb( '<p id="breadcrumbs">', '</p>' );
-				}
-				?>
+				<h2><?php echo get_queried_object()->user_nicename ?></h2>
```

Comment:
We can make this code a bit more clear with the previously suggested context variables.
```suggestion
				<h2><?php echo $author_nicename ?></h2>
```